content/kubermatic/main/tutorials-howtos/administration/presets/_index.en.md

```diff
@@ -236,3 +236,8 @@ kubectl edit preset my-preset
 
 This will open the editor configured for `kubectl edit` and let you update the YAML specification.
 It is possible to update any fields already present or add/remove provider sections from the `Preset`.
+
+## Limiting Permissions for Credentials used in Presets
+
+For selected infrastructure providers it makes sense to limit the permissions of the credentials to protect against access creep.
+For these cases, we have compiled a [list of minimum permissions]({{< relref "../../../../../kubeone/main/architecture/requirements/machine-controller/" >}}) required for different providers.
```

Comment:
@SimonTheLeg let's discuss your point here:

> @embik This PR covers a couple of topics simultaneously (for all the details see the attached issue), but there is one thing I wanted your opinion on: The best way I found to make the connection between the KKP installation guide and our existing reference on required [infrastructure permissions](https://docs.kubermatic.com/kubeone/v1.5/architecture/requirements/machine-controller/), was to include the presets guide as a Next steps in the installation guide and then ammend the presets guide with a link to the infra permissions reference. This was the best flow I could figure out. Since you have done the rewrite of the install guide: How do you feel about this?

I like the flow you described, however I think linking to KubeOne is not a good idea here. The permissions documented in KubeOne's machine-controller section refer to what is needed to create VMs on respective cloud providers. But KKP needs permissions beyond that for setting up cloud resources per user cluster. So these permissions are likely incomplete. Have you tried them out for working with KKP?

Reply:
So I have not tried them out personally, but asked in the PS channel what we typically use for customers. Maybe @dharapvj can chime in on this: When we talked about it on Slack, is the documentation linked only valid for Kubeone or do we also use this for KKP?

Reply:
that's a good point.. I would imagine that these same permissions work in both kubeone and kkp since both use machine-controller to create machines. But I have lesser exposure to KKP deployments in AWS in my daily work

Reply:
That's my point, KKP does use machine-controller to create machines, but it also creates the resources needed to provision those machines. All the stuff happening in https://github.com/kubermatic/kubermatic/tree/main/pkg/provider/cloud, effectively. KubeOne would need some of these permissions for Terraform as well (but are they the same? We don't know), but the linked page is explicitly only for machine-controller, so it should be considered a subset of needed permissions.

Reply:
Agreed.

Then only way out .. probably is to start with absolute minimal permission IAM user and go on adding permissions until everything has panned out.

Also - see this file - which as some interesting things in it..
https://github.com/kubermatic/kubermatic/blob/main/pkg/provider/cloud/aws/policy.go